crates/oxpinyin-python/tests_py/test_engine.py

```
@@ -0,0 +1,201 @@
+    with make_engine() as engine:
+        assert engine.lookup("nihao")[0].text == "你好"
+    # close() is also directly callable and idempotent-friendly
+    make_engine().close()
+
+
+def test_shared_engine_is_thread_safe(make_engine):
+    engine = make_engine()
+    expected = [c.text for c in engine.lookup("nihao")]
+    errors = []
+
+    def worker():
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `worker`

Add return type annotation: `None`

(ANN202)